rapids-cmake/cpm/nvbench.cmake

```diff
@@ -29,12 +29,16 @@ across all RAPIDS projects.
 
 .. code-block:: cmake
 
-  rapids_cpm_nvbench( [BUILD_EXPORT_SET <export-name>] )
+  rapids_cpm_nvbench( [BUILD_EXPORT_SET <export-name>]
```

Comment:
Installing nvbench has rammifications for dowstream users. Primarily the supported set of cuda architectures are baked into the nvbench headers/library. When installed using it from a project with a different set of cuda architectures can cause problems ( wrong results / odr violations ). ( https://github.com/NVIDIA/nvbench/issues/21 )

Instead of risking this happening ( install a version built for native, used by consumer building for all ) we don't install nvbench at all.

Reply:
Cool, I figured it was arch related. Thanks for the context.